workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/androidx/WorkflowAndroidXSupport.kt

```diff
@@ -15,6 +15,17 @@ import kotlin.reflect.cast
  * Namespace for some helper functions for interacting with the AndroidX libraries.
  */
 public object WorkflowAndroidXSupport {
+  /**
+   * Returns the [LifecycleOwner] managing [context].
+   *
+   * @throws IllegalArgumentException if [context] is unmanaged
+   */
+  @WorkflowUiExperimentalApi
+  public fun lifecycleOwnerFromContext(context: Context): LifecycleOwner =
+    requireNotNull(context.ownerOrNull(LifecycleOwner::class)) {
```

Comment:
I'm guessing this isn't your API but funny that we have "ownerOrNull"  instead of "requireOwner"

Reply:
Actually, this makes me think, should this turn into `Context.requireLifecycleOwner` ?

Reply:
Very consistent convention we switched to ages ago. `orNull` is the weird case. If you don't specify it, non-null is implied.

Reply:
`require*` is so Java.

Reply:
`should this turn into Context.requireLifecycleOwner`

We've left these methods a bit inconvenient on purpose. Feature devs shouldn't need them, having them autocompolete off of `Context` would be more advertising than we really want.

Reply:
`<x>OrNull` is the Kotlin style - see e.g., Result - https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-result/get-or-null.html